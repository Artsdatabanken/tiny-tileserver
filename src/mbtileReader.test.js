const mbTileReader = require("./mbtileReader");

test("readTile", () => {
  return mbTileReader
    .readTile("./testdata/pbf.mbtiles", 3, 4, 2)
    .then(tile => expect(tile).toMatchSnapshot());
});
