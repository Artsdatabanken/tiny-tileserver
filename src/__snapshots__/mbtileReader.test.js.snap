// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`readTile 1`] = `
Object {
  "data": Array [
    31,
    139,
    8,
    0,
    0,
    0,
    0,
    0,
    2,
    3,
    147,
    138,
    173,
    96,
    226,
    226,
    112,
    244,
    143,
    55,
    52,
    212,
    53,
    49,
    213,
    104,
    80,
    144,
    98,
    73,
    44,
    74,
    77,
    148,
    98,
    73,
    206,
    79,
    73,
    149,
    98,
    201,
    75,
    204,
    77,
    85,
    226,
    230,
    226,
    52,
    180,
    176,
    180,
    48,
    182,
    52,
    50,
    49,
    83,
    226,
    66,
    168,
    86,
    98,
    227,
    98,
    113,
    202,
    207,
    206,
    19,
    146,
    148,
    96,
    22,
    98,
    99,
    96,
    96,
    100,
    100,
    98,
    82,
    226,
    229,
    124,
    196,
    126,
    71,
    76,
    74,
    65,
    194,
    80,
    137,
    89,
    147,
    31,
    0,
    216,
    169,
    196,
    67,
    95,
    0,
    0,
    0,
  ],
  "type": "Buffer",
}
`;
